tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs

```diff
@@ -47,8 +47,7 @@ public async Task TestMultiLangImport()
             // use LG file as entrance
             var lgResourceGroup = LGResourceLoader.GroupByLocale(resourceExplorer);
 
-            var resource = resourceExplorer.GetResource("a.en-US.lg") as FileResource;
-            var generator = new TemplateEngineLanguageGenerator(resource.FullName, lgResourceGroup);
+            var generator = new TemplateEngineLanguageGenerator(resourceExplorer.GetResource("a.en-US.lg").FullName, lgResourceGroup);
```

Comment:
>TemplateEngineLanguageGenerator [](start = 32, length = 31)

Isn't this an opportunity to make the LG fully resource based?  Why not add in a constructor which takes a resource?

Reply:
updated, add a new method for this